example/demo/tsconfig.json

```diff
@@ -5,10 +5,14 @@
 		"lib": ["es2021"],
 		"module": "es2022",
 		"moduleResolution": "Bundler",
-		"types": ["@cloudflare/workers-types"]
+		"types": ["@cloudflare/workers-types"],
+		"jsx": "react-jsx",
+    "jsxImportSource": "@final-ui/react"
```

Comment:
```
"jsx": "react-jsx",
    "jsxImportSource": "@final-ui/react"
```
is all you have to set to enable JSX support. Will be currently manual, but in next PR we should create a preset.

Reply:
Right, makes sense